lib/omniauth/strategies/bitbucket.rb

```diff
@@ -21,18 +22,22 @@ class Bitbucket < OmniAuth::Strategies::OAuth
 
       info do
         {
-          :name => "#{raw_info['first_name']} #{raw_info['last_name']}",
-          :avatar => raw_info['avatar'],
+          :name => raw_info['nickname'],
+          :avatar => raw_info['links']['avatar']['href'],
```

Comment:
why are we returning the href instead of the avatar here?

Reply:
I think `{'avatar' => 'http://asdf'}` is now `{'links' => {'avatar' => {'href' => 'http://asdf'}}}`